wrf/utils.py

```diff
@@ -922,8 +989,11 @@ def extract_column_from_wrfdata(fpath, coords,
     return xn
 
 
-def combine_towers(fdir, restarts, simulation_start, fname, return_type='xarray', structure='ordered',
```

Comment:
@phawbeck These are the changes I made a couple weeks ago, including: better handling of the input parameters, removal of the df return type, removal of the ambiguous/redundant SIMULATION_START_TIME attribute, and general code cleanup.

Reply:
Perfect - they look how I pictured them when we spoke.